specifications/api/swagger.yaml

```diff
@@ -892,7 +892,7 @@ paths:
       parameters:
         - *parametersWalletId
         - <<: *parametersBody
-          schema: *parametersPostTransaction
+          schema: *PostTransactionData
```

Comment:
Following naming schema from https://github.com/input-output-hk/cardano-wallet/blob/5c7d690e5d994c931e5b437c5bdbaf222f6dc675/src/Cardano/Wallet/Api.hs#L88 

Although vice versa naming would also be ok (ie, that we are following naming suggestions from swagger spec in our haskell datatypes)